openshift/4.0/templates/network-security-policy/service-account-to-k8s-api.yaml

```diff
@@ -1,28 +0,0 @@
-kind: Template
```

Comment:
This is no longer needed?

Reply:
this is an egress policy which isn't supported in KNPs at the moment.